TaskList/app/views/layouts/_task_printout.html.erb

```diff
@@ -33,7 +33,7 @@
           <%= button_to button_path, method: :delete, data: { confirm: "Are you sure?" }, class: "btn info-btn btn-sm", title: "Delete Task"  do %>
             <span class="glyphicon glyphicon-remove-sign" aria-hidden="true" aria-label="Delete Button"></span>
           <% end %>
-          <%= button_to button_path, method: :delete, class: "btn info-btn btn-sm", title: "Toggle Completed" do %>
+          <%= button_to "#{button_path}/completed", method: :patch, class: "btn info-btn btn-sm", title: "Toggle Completed" do %>
```

Comment:
This comment is more about line 32 (where `button_path` is set)... I think it might be better to move that variable definition into the controller. You're already using the `@action_url` variable in the controller which has direct path elements, so I think it would be better to consolidate all of the URL variables into one place.